.gitignore

```diff
@@ -1,16 +1,15 @@
-# Intellij files
-#IDE
-.idea/
-**/*.iml
+target
 
-# jenkins stuff
-work/
+# mvn hpi:run
+work
 
-# maven stuff
-.mvn/maven.config
```

Comment:
This file needs to be checked in for [Incrementals](https://github.com/jenkinsci/incrementals-tools).